extern/jsoncpp/src/lib_json/json_reader.cpp

```diff
@@ -1,5 +1,5 @@
-#include "json/reader.h"
-#include "json/value.h"
```

Comment:
This should not be a problem whatsoever, but I am wondering why this was changed initially.